.github/workflows/kind-e2e.yaml

```diff
@@ -97,8 +96,8 @@ jobs:
         # Configure registry for KinD.
         containerdConfigPatches:
         - |-
-          [plugins."io.containerd.grpc.v1.cri".registry.mirrors."${{ matrix.registry-url }}"]
-            endpoint = ["http://${{ matrix.registry-url }}"]
+          [plugins."io.containerd.grpc.v1.cri".registry.mirrors."$REGISTRY_NAME:$REGISTRY_PORT"]
```

Comment:
is this intentional? I think before this evaluated to ` [plugins."io.containerd.grpc.v1.cri".registry.mirrors."registry.local:5000/knative"]` whereas now it's `[plugins."io.containerd.grpc.v1.cri".registry.mirrors."registry.local:5000"]`

Reply:
```suggestion
          [plugins."io.containerd.grpc.v1.cri".registry.mirrors."$REGISTRY_URL"]
```

Reply:
oh wait, no, Im wrong! you didn't add a REGISTRY_URL, that was on my branch :D